nixpkgs_review/github.py

```diff
@@ -16,6 +24,11 @@ def get(self, path: str) -> Any:
             req.add_header("Authorization", f"token {self.api_token}")
         return json.loads(urllib.request.urlopen(req).read())
```

Comment:
I would suggest to move the code above to `_request(..., method)` and add `Request(url, method=method)`. Then you can introduce a function called `get()` that calls `_request(..., method="GET")` and one for `post()` that calls `_request(..., method="POST")`

Reply:
Done